sdk/monitor/monitor-query/test/public/shared/testShared.ts

```diff
@@ -117,14 +107,17 @@ export function getAppInsightsConnectionString(mochaContext: Pick<Context, "skip
 }
 
 function getRequiredEnvVar(mochaContext: Pick<Context, "skip">, variableName: string): string {
-  if (!env[variableName]) {
+  const envVar = env[variableName];
+
+  if (!envVar) {
     console.log(
       `TODO: live tests skipped until test-resources + data population is set up (missing ${variableName} env var).`
     );
     mochaContext.skip();
+    // NOTE: This function should be throwing
```

Comment:
Just wondering, what do you mean by this? If the function should throw we can use the `assertEnvironmentVariable` helper exposed by the recorder instead of defining this function.

Reply:
This was a note that came out during a sync, I forgot to update the function to add a `Throw new Error(...` in case the environment variable is not defined (right now it is just skipping it).

Can you provide more info about the `assertEnvironmentVariable` helper and how it could be used?

Reply:
We have an [`assertEnvironmentVariable` function](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/test-utils/recorder/src/utils/utils.ts#L355-L362) exported from the recorder package which either returns the value of the environment variable or throws an error if it's not set. Maybe you could use that instead of the helper that's here at the moment (and remove old the helper entirely).